modules/ztp-configuring-spoke-at-remote-site.adoc

```diff
@@ -0,0 +1,162 @@
++
+[source,terminal]
+----
+$ vi /etc/resolv.conf
+----
+
+.. Add the line.
++
+[source,terminal]
+----
+nameserver 192.168.7.10
+----
+
+. Configure a static IP on the connected laptop:
+
+.. Determine the name of the laptop's network interface card (NIC) as follows.
++
+[source,terminal]
+----
+$ ip addr
+----
++
+[NOTE]
+====
+Look for the NIC name starting with the letter `e`.
+====
+
+.. Edit the NIC's network configuration.
++
+[source,terminal]
+----
+$ vi /etc/sysconfig/network-scripts/ifcfg-eth0
+----
++
+[NOTE]
+====
+Here `eth0` is the network card name, and it can be different for different computers.
+====
+
+.. Add or modify the configuration below:
++
+[source,terminal]
+----
+BOOTPROTO=static
+IPADDR=192.168.7.21
+NETMASK=255.255.255.0
+GATEWAY=192.168.0.1
+DNS1=8.8.8.8
+DNS2=8.8.4.4
+----
++
+[NOTE]
+====
+Use any IP in the range `192.168.7.20 - 192.168.7.150`.
+====
+
+. Restart the network services:
++
+[source,terminal]
+----
+$ systemctl restart NetworkManager
+----
+
+. Open a browser and log in to the edge cluster configuration user interface at the following URL with the supplied `kubeadmin` `username` and `password`:
++
+[source,terminal]
+----
+$ https://edge-cluster-setup.foobar-edge-cluster.domain.com
+----
++
+[NOTE]
+====
+This `kubeadmin` username and password was created at the factory and should have been supplied to you. Only one user is initially created.
+====
+
+. Click **Continue**.
+
+. Step through the screens to complete the initial setup.
+
+.. In the first two screens create a new user account by entering a `username` and `password` when prompted.
++
+[NOTE]
+====
+This new user account is granted `cluster-admin` privileges and should be used rather than the factory created `kubeadmin` account.
+====
+
+.. In the **API** screen assign the IP address that will be used for API traffic. A default value is assigned but you are free to update this.
+
+.. In the **Ingress** screen assign the IP address that will be used for new routes and traffic managed by the ingress controller. A default value is assigned but you are free to update this.
+
+.. Optional: Under Domain create unique URLs for the setup and console URLs for your edge cluster.
+
+.. Click **Download** in the **SSH** screen and download the edge cluster private SSH key.
++
+[NOTE]
+====
+You need this to access the nodes of the edge cluster.
+====
+
+.. Click **Finish setup**.
++
+When complete the screen switches to:
++
+.Edge cluster setup
+image::ztp-edge-cluster-setup1.png[Edge cluster setup]
+
+.. Selecting **OpenShift console** brings you direct to the web console.
+
+.. Under **Settings** you have the option to change the values of the **API address**, **Ingress address** and the previously configured **Domain**.
++
+[NOTE]
+====
+At this stage you will not be prompted for a username and password as you are already logged in as `kubeadmin`.
+====
+
+.. Click **Log out** in the top right hand corner.
+
+.Verification and final setup
```

Comment:
Only ".Verification" is an approved heading in the mod docs templates. You should either adjust it to be that, or if it's actually including a required step, the probably just get rid of that and allow it to be part of the main procedure